src/systems/menu_impl.cpp

```diff
@@ -406,20 +408,26 @@ static void draw_help(Widget widget) {
             draw::debug_text(BUTTON_START + 3 * BLOCK_WIDTH, Y_HEIGHT, draw::WHITE, ": Reset");
             break;
         }
-        default: {
+        case WidgetType::Text:
+        case WidgetType::ColoredText:
+        case WidgetType::Header:
+        case WidgetType::Separator:
+        case WidgetType::FloatView:
+        case WidgetType::HideableGroupWidget:
+        case WidgetType::Custom: {
```

Comment:
I think we can make an exception to the "don't include default cases" rule here when there's plenty of options you'd never want to handle :)

Reply:
Gotcha, I'll put this default case back